jdbc/src/test/java/com/interface21/jdbc/core/JdbcTemplateTest.java

```diff
@@ -1,5 +1,91 @@
 package com.interface21.jdbc.core;
 
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import javax.sql.DataSource;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
 class JdbcTemplateTest {
 
+    private DataSource dataSource;
+    private Connection connection;
+    private PreparedStatement preparedStatement;
+    private PreparedStatementSetter preparedStatementSetter;
+    private RowMapper rowMapper;
+    private ResultSet resultSet;
+    private JdbcTemplate jdbcTemplate;
+
+    @BeforeEach
+    void setUp() throws Exception {
+        dataSource = mock(DataSource.class);
+        connection = mock(Connection.class);
+        preparedStatement = mock(PreparedStatement.class);
+        preparedStatementSetter = mock(PreparedStatementSetter.class);
+        rowMapper = mock(RowMapper.class);
+        resultSet = mock(ResultSet.class);
+
+        when(dataSource.getConnection()).thenReturn(connection);
+        when(connection.prepareStatement(any(String.class))).thenReturn(preparedStatement);
```

Comment:
```suggestion
        when(connection.prepareStatement(anyString())).thenReturn(preparedStatement);
```
그냥 참고사항이에요~ 이런 메서드도 있습니다.

Reply:
👍👍